client-ui/src/app/components/dashboard/dashboard.component.ts

```diff
@@ -87,119 +81,111 @@ export class DashboardComponent implements OnInit, AfterViewInit {
   } as NgrxValueConverter<Date, number>
 
   ngOnInit(): void {
-    this.store
-      .select('router', 'state', 'queryParams')
-      .subscribe((queryParams: any) => {
-        if (!_.isEmpty(queryParams)) {
-          this.timestampStart = queryParams.timestampStart
-          this.timestampEnd = queryParams.timestampEnd
-          this.asAtTime = this.asAtTime = Number(queryParams.asAtTime) || this.asAtTime
-          this.offset = Number(queryParams.offset)
-          this.sortName = queryParams.sortName
-          this.sortDirection = queryParams.sortDirection
-          this.searchTerm = queryParams.searchTerm
-          this.updateDateRange(moment(Number(queryParams.timestampStart)).toDate(), moment(Number(queryParams.timestampEnd)).toDate())
-          this.store.dispatch(new ExecutionEventsActions.Get(queryParams))
-          this.store.dispatch(new DashboardFormActions.InitializeForm({ minDate: this.timestampStart, maxDate: this.timestampEnd }))
-        } else {
-          this.store.dispatch(new ExecutionEventsActions.GetDefault({}))
-        }
-      })
-
-    this.store
-      .select('dashboardForm', 'dashboardFilters', 'value', 'dateRange')
-      .pipe(
-        filter(state => state != null)
-      )
-      .subscribe(dateRange => {
-        this.updateDateRange(moment(dateRange[0]).toDate(), moment(dateRange[1]).toDate())
-      })
+    this.subscribtions.push(
+      this.store.select('router', 'state', 'queryParams')
+        .subscribe((queryParams: any) => {
+          if (!_.isEmpty(queryParams)) {
+            this.queryParams = queryParams
+            if (queryParams.timestampStart != 0) {
+              this.liveData = false
```

Comment:
As discussed we need to clarify if taking the state outside of the observable and managing it at the component level directly is Ok from the perspective best practices. Otherwise LGTM